pages/common/llc.md

```diff
@@ -0,0 +1,16 @@
+# llc
+
+> Compiles LLVM Intermediate Representation or bitcode to target-specific assembly language.
+> More information: <https://www.llvm.org/docs/CommandGuide/llc.html>.
+
+- Compile a bitcode or IR file to an assembly file with the same base name:
+
+`llc {{path/to/file.ll}}`
+
+- Enable all optimizations and output assembly to a specific file:
```

Comment:
The page has a few examples, can we split this into 2 exmaples to be more clear about `--output` and `-03`?